tests/create_guardduty_org_admin_account/main.tf

```diff
@@ -0,0 +1,38 @@
+# AWS provider account for the AWS organization
+provider "aws" {
+  region  = "us-east-1"
+  profile = "plus3it-management" # Profile must exist in your .aws/config
+}
+
+# AWS provider account for the GuardDuty org administrator account
+provider "aws" {
+  region  = "us-east-1"
+  alias   = "guardduty_administrator"
+  profile = "plus3it-member" # Profile must exist in your .aws/config
```

Comment:
```suggestion
  profile = "awsalternate" # Profile must exist in your .aws/config
```

Reply:
I used these profile names to make it clear which account was being used for what.  it was confusing for me to understand this using the more generic names.  I am trying to make this easier for the next person to understand.  I'll make this change if you really want me to, but I feel like we will lose some understandability.

Reply:
Change made.  I would like to read about the test framework of the terraform aws provider.  I googled it but did not find much.  Can you please suggest a resource?